config/marathon_event_exporter.schema.exs

```diff
@@ -0,0 +1,85 @@
+@moduledoc """
+A schema is a keyword list which represents how to map, transform, and validate
```

Comment:
I'm wondering where all these docs come from... is there a command you ran to generate this file?

Reply:
Yes, https://hexdocs.pm/conform/getting-started.html is the relevant documentation for that.

I edited after it was generated, though.